html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm

```diff
@@ -566,7 +573,7 @@ sub showLogin : Private {
     $c->stash(
         template        => 'login.html',
         username        => $request->param_encoded("username") ,
-        null_source     => is_in_list( $SELFREG_MODE_NULL, $guestModes ),
+        null_source     => is_in_list( $SELFREG_MODE_NULL, $guestModes ) || is_in_list( $SELFREG_MODE_KICKBOX, $guestModes ),
```

Comment:
Yeah I know but if I change the name, I need to change it in the templates and as they are upgraded like :hankey: it will mean manual work.

Either way we get :hankey: